js/bittrex.js

```diff
@@ -1473,9 +1535,17 @@ module.exports = class bittrex extends Exchange {
         //         "clientWithdrawalId": "string (uuid)"
         //     }
         //
-        const id = this.safeString2 (transaction, 'id', 'clientWithdrawalId');
+        const id = this.safeString (transaction, 'id', 'clientWithdrawalId');
```

Comment:
should we use **safeString2** or remove 'clientWithdrawalId' ? currently we're setting 'clientWithdrawalId' as the default value